tests/sys/fs/fusefs/xattr.cc

```diff
@@ -119,6 +121,33 @@ void expect_setxattr(uint64_t ino, const char *attr, const char *value,
 
 };
 
+class Xattr_7_32: public FuseTest {
+public:
+	virtual void SetUp() {
+		m_kernel_minor_version = 32;
+		FuseTest::SetUp();
+	}
+	
+void expect_setxattr_7_32(uint64_t ino, const char *attr, const char *value,
+	ProcessMockerT r)
+{
+	EXPECT_CALL(*m_mock, process(
+		ResultOf([=](auto in) {
+			const char *a = (const char*)in.body.bytes +
+				FUSE_COMPAT_SETXATTR_IN_SIZE;
+			const char *v = a + strlen(a) + 1;
+			return (in.header.opcode == FUSE_SETXATTR &&
+				in.header.nodeid == ino &&
+				in.body.setxattr.size == (strlen(value) + 1) &&
+				in.body.setxattr.flags == 0 &&
```

Comment:
You shouldn't check setxattr.flags right here, because it's undefined.  It actually overlaps with attr.